### PR TITLE
Scrape events

### PR DESCRIPTION
**Pull request by Emmanuel96**

### CODE REVIEW

File: src/server/scraper/scrape.py [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+import time
+import random
+
+load_dotenv()
+
+def storeEvents(eventListings, cursor, connection):
+  linkPrefix = "https://www.meetup.com"
+
+  for event in eventListings:
+    cuid = generateCuid();
```
Comment (foresthpark) [inline src/server/scraper/scrape.py:17]: cuid is auto generated on the database as an id so this is not needed. Please change later

**Review by foresthpark — APPROVED** [review]

**Comment by foresthpark** [comment]
@Emmanuel96 I did a bit of research into Python scripts running on Vercel and the problem seems to be with loading the env files onto the script. Currently there seems to be no way to do it. I was mistaken when I said the script will run on Vercel. I apologize. 

If you can deploy it onto Heroku(or anything else), I can set up a cron job on this app that will just ping the Heroku endpoint. Just allow the endpoint to return a simple success/fail message and I can go from there.

**Comment by foresthpark** [comment]
@Emmanuel96 I have also added some comments onto the scraper script for you to review. Thanks.

**Comment by Emmanuel96** [comment]
@foresthpark That's okay. I would have been very surprised if it was possible. 

We have the option of making a GitHub action. It's a better choice because it'll work regardless of our hosting platform. 

I'll create that and re-commit.

**Comment by Emmanuel96** [comment]
and I've already started with making the cron job on this app; I don't think there's any need for a ping. 

The cron job on this app can run daily, to help keep our planetscale db alive and working. 

And the scraper can scrape once every month.

**Comment by Emmanuel96** [comment]
> @Emmanuel96 I have also added some comments onto the scraper script for you to review. Thanks.

I also cannot see the comments you're referring to for what ever reason.

**Pull request merged**